jbpm-wb-dashboard/jbpm-wb-dashboard-client/src/main/java/org/jbpm/dashboard/renderer/client/panel/ProcessDashboard.java

```diff
@@ -337,19 +339,27 @@ public void tableCellSelected(String columnId,
         Integer processInstanceStatus = Double.valueOf(ds.getValueAt(rowIndex,
                                                                      COLUMN_PROCESS_STATUS).toString()).intValue();
 
-        openProcessDetailsScreen();
-        setupDetailBreadcrumb(i18n.processDashboardName(),
-                              i18n.ProcessInstanceBreadcrumb(processInstanceId),
-                              PROCESS_INSTANCE_DETAILS_SCREEN);
-
-        instanceSelectionEvent.fire(new ProcessInstanceSelectionEvent(
-                deploymentId,
-                processInstanceId,
-                processDefId,
-                processDefName,
-                processInstanceStatus,
-                true,
-                serverTemplateSelectorMenuBuilder.getSelectedServerTemplate()));
+        specManagementService.call((ServerTemplate serverTemplate) -> {
+            if (serverTemplate.getContainerSpec(deploymentId).getStatus().equals(KieContainerStatus.STARTED)) {
+                openProcessDetailsScreen();
+                setupDetailBreadcrumb(i18n.processDashboardName(),
+                                      i18n.ProcessInstanceBreadcrumb(processInstanceId),
+                                      PROCESS_INSTANCE_DETAILS_SCREEN);
+
+                instanceSelectionEvent.fire(new ProcessInstanceSelectionEvent(
+                        deploymentId,
+                        processInstanceId,
+                        processDefId,
+                        processDefName,
+                        processInstanceStatus,
+                        true,
+                        serverTemplateSelectorMenuBuilder.getSelectedServerTemplate()));
+            } else {
+                displayWarningNotification(i18n.processDetailsNotAvailable());
```

Comment:
I think we could be more specific about the issue, stating something like: "Process details not available, deployment container X is not started", wdyt?

Reply:
+1